src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_IsByRefLike.cs

```diff
@@ -730,18 +753,10 @@ class Test
                 var type = module.ContainingAssembly.GetTypeByMetadataName("Test").GetTypeMember("S1");
                 Assert.True(type.IsByRefLikeType);
 
-                var accessibility = Accessibility.Public;
                 var attributes = type.GetAttributes();
-                Assert.Equal(2, attributes.Count());
-
-                var assemblyName = module.ContainingAssembly.Name;
+                Assert.Equal(1, attributes.Count());
 
-                var attributeType = attributes[0].AttributeClass;
-                Assert.Equal("System.Runtime.CompilerServices.IsByRefLikeAttribute", attributeType.ToDisplayString());
-                Assert.Equal(assemblyName, attributeType.ContainingAssembly.Name);
-                Assert.Equal(accessibility, attributeType.DeclaredAccessibility);
-
-                var attribute = attributes[1];
+                var attribute = attributes.Single();
```

Comment:
>attributes.Single(); [](start = 32, length = 20)

`Single()` throws if there isn't exactly one element. So I think the above assert is redundant.